src/_ert_forward_model_runner/job.py

```diff
@@ -179,7 +179,7 @@ def ensure_file_handles_closed():
 
         max_memory_usage = 0
         while exit_code is None:
-            memory_rss = _get_rss_for_processtree(process)
+            (oom_score, memory_rss) = _get_oom_score_and_rss_for_processtree(process)
```

Comment:
memory_rss is the primary target, and oom_score is merely a bonus (really, it is a maybe-nice-to-have, maybe it should be the other way around.

Reply:
swapped them and changed name of the function to reflect it